### DISCUSSION TITLE
Horizontal fifths?

### DISCUSSION THREAD

**Discussion by willhaslett-stripe** [rxhanson/Rectangle#1308] [General]
Hi. I have a very wide monitor and my best setup is to have 5 windows across the screen horizontally. It doesn't look like this is supported. Is there a way to split into 5ths with Rectangle?

**Comment by rxhanson** [comment]
At the moment, there is not. I'm not opposed to merging a pull request that adds this as a hidden setting, like what was done for 9ths.

Alternatively, you could use the custom shortcut builder in [Rectangle Pro](https://rectangleapp.com/pro) (paid):

![image](https://github.com/rxhanson/Rectangle/assets/13651296/e3c2fede-5146-4a04-b71f-f69e43089937)

**Reply by willhaslett-stripe** [reply]
Random feature idea. An LLM could prolly do a good job of just implementing a layout that a user describes using natural language!

![Screenshot 2023-12-14 at 1 27 56 PM](https://github.com/rxhanson/Rectangle/assets/88849903/56d848c3-aff2-4bb7-ae40-b95917969029)

**Reply by rxhanson** [reply]
Ah, very nice. I actually haven't thought of that before. You could probably hand it the exported json and get it to rewrite it with a new one, too if it can understand the existing json.

That's a great idea though: a good API that an LLM can use. Adding it to the backlog.

**Comment by kunanyi** [comment]
An option for Fifths would be great.   (although in my case I'd like a window size 4/5th wide, just so I can see the mounted drives on my desktop and not much else)

Looking into Pro, but atm I don't think I need much more than 5ths.  But if I were to buy a window manager this would certainly be it!

**Comment by MacDada** [comment]
I would also like to request adding "Fifths".

I've already tried to set it up via `Custom Size & Position`:
![Screenshot 2024-03-26 at 18 32 11](https://github.com/rxhanson/Rectangle/assets/136261/9f23e808-23c1-41cc-a82a-9bf8499701ef)

The problem is: I would have to define all 5 fifths, with 5 separate shortcuts to invoke them.

What I would like is the same behavior I have with "Forths" or "Sixths": when I repeat one shortcut, it cycles through the "n-ths". So, with my first shortcut hit, I get the first "n-th", when hit again I get second "n-th", etc. This way I actually use one shortcut to position the window in the place I want.

**Reply by rxhanson** [reply]
Rather than using the + button in the top right, you'll want to add each one as a repetition, using the + button that appears on hover over your first fifth config.

With a better UI for custom shortcuts, perhaps it wouldn't be as much of a desire to have a fifths config. I have UI improvements here on my backlog.

**Reply by MacDada** [reply]
> Rather than using the + button in the top right, you'll want to add each one as a repetition, using the + button that appears on hover over your first fifth config.

Awesome, it works! Thanks!

![Screenshot 2024-03-27 at 01 51 45](https://github.com/rxhanson/Rectangle/assets/136261/e6505a94-8218-4b32-8d9c-477dcc80c957)
